### ISSUE TITLE
Npm version 8.6.0 dependency install problems

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace-WebGuiFrontend#84]
Installing dependencies with npm version 8.6.0 (the one you get with Node at the time of writing) leads to errors. See image below

Did not look into detail why. For now, a working workaround is to use npm version 8.5.5. To downgrade, run `npm install -g npm@8.5.5`

![image](https://user-images.githubusercontent.com/8808894/162695338-29e888ca-a748-45ab-b70c-149414bd1b3a.png)

**Comment by alexanderbock** [comment]
:'(  Using Node/NPM straight out of the box was nice while it lasted

**Comment by WeirdRubberDuck** [comment]
> :'( Using Node/NPM straight out of the box was nice while it lasted

Indeed... 😅 But hopefully it's just a matter of updating one more dependency
